src/Simulation/QCTraceSimulator/DepthCounter.cs

```diff
@@ -93,7 +103,8 @@ public void OnOperationEnd(object[] returnedQubitsTraceData)
             double[] metrics =
                 StatisticsRecord( 
                     Depth : operationEndTime - opRec.MaxOperationStartTime,
-                    StartTimeDifference: opRec.MaxOperationStartTime - opRec.MinOperationStartTime );
+                    StartTimeDifference: opRec.MaxOperationStartTime - opRec.MinOperationStartTime,
+                    Width: qubitAvailabilityTime.GetMaxQubitId() - opRec.MaxQubitIdAtStart );
```

Comment:
I am confused here. If the QubitManager was configured to reuse qubits it's possible that the operation reuses all of the qubits it allocates, meaning the Width will be calculated as zero. But this doesn't mean the operation's width is zero, right?

Reply:
Correct about calculated with = 0. This is the ongoing discussion about the definition of Width, AddedWidth and how they should be documents.